time stamp_robot_update
time action_taken_time
string robot_action_taken
string robot_belief_state
string robot_real_state

string immediate_reward # from prev step
string total_disc_reward
